inc/packages/namespace.php

```diff
@@ -98,7 +99,7 @@ function get_did_document( string $id ) {
 	if ( is_wp_error( $document ) ) {
 		return $document;
 	}
-	set_site_transient( $id, $document, CACHE_LIFETIME );
+	wp_cache_set( $id, $document, 'did', CACHE_LIFETIME );
```

Comment:
Not for this PR, but I'm wondering if we should actually be caching the HTTP response in the `::fetch_document()` method rather than the document object here. That would mean anything calling `::fetch_document()` could benefit from the cache, and would make it consistent with the metadata cache, which caches the HTTP response, not the document object.